config/keys.go

```diff
@@ -65,5 +71,27 @@ func LoadSigningKeys() (*SigningKeys, error) {
 		}
 		return nil, err
 	}
+
+	if err := validateKeys(&config); err != nil {
+		return nil, err
+	}
+
 	return &config, nil
 }
+
+func validateKeys(config *SigningKeys) error {
+	keys := config.Keys
+	uniqMap := make(map[string]struct{}, len(keys))
```

Comment:
We do have a [set](https://github.com/notaryproject/notation-go/blob/main/internal/container/set.go) data structure implemented in notation-go.